src/type/definition.js

```diff
@@ -674,6 +696,23 @@ export class GraphQLObjectType {
     return this._interfaces;
   }
 
+  toConfig(): {|
+    ...GraphQLObjectTypeConfig<*, *>,
```

Comment:
There's definitely another project for someone of figuring out how to get rid of all these `*` flow types (which are about as bad as `any`). Not this PR though.